[CN-1110]: Add docs for custom map attribute config (#153)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/map-configuration.adoc

|indexes
|Indexes to be created for the map data.

|attributes

> |custom attributes to use the Predicates API to query fields

Reviewer: ```suggestion
|Custom attributes for field queries when using the Predicates API
```
